pkg/api/aim/apps.go

```diff
@@ -2,18 +2,29 @@ package aim
 
 import (
 	"fmt"
+	"net/http"
 
 	"github.com/gofiber/fiber/v2"
 	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
 	"gorm.io/gorm"
 
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api"
+	"github.com/G-Research/fasttrackml/pkg/common/middleware/namespace"
 	"github.com/G-Research/fasttrackml/pkg/database"
 )
 
 func GetApps(c *fiber.Ctx) error {
+	ns, err := namespace.GetNamespaceFromContext(c.Context())
+	if err != nil {
+		return api.NewInternalError("error getting namespace from context")
+	}
+	log.Debugf("getApps namespace: %s", ns.Code)
+
```

Comment:
it seems like we could do this "extract from context" in a middleware or something so it is not repeated in every single controller function

Reply:
To clarify, why even put it in the context if we extract it right away in the controller function? Couldn't we just as easily call a function here with `fiber.Ctx.Params("ns")` which returns the `ns.ID` or `Namespace` struct?